guide/problems/index.md

```diff
@@ -3,6 +3,27 @@ title: Common Problems
 ---
 We try and make the REST API as easy as possible, but APIs are hard. Here's a collection of common issues we run into.
 
+* TOC
+{:toc}
```

Comment:
This is indeed a thing, it is one of my favorite things about Jekyll / GH pages actually:

![image](https://cloud.githubusercontent.com/assets/442115/18518389/c081bfba-7a6d-11e6-9a96-81d28f2eda27.png)

Updated the backticks!